packages/xo-server/src/vhd-merge.js

```diff
@@ -424,69 +424,66 @@ export class Vhd {
       : fromEvent(data.pipe(stream), 'finish')
   }
 
+  async freeFirstBlockSpace (spaceNeededBytes) {
```

Comment:
I feat this method is not enough if use it both for the BAT and parent locators.
Don't you think there are risks of collision?

Reply:
What's your opinion on this?

Reply:
I am sorry, I don't understand the issue